testsuite/mlir/mlir-parse/dummy/out.mlir

```diff
@@ -1,10 +1,20 @@
 
 
 module {
-  func @bar(%arg0: !verona.U64) -> !type<"U64 & imm"> {
-    %0 = verona.foo %arg0 : !verona.U64
-    %1 = "verona.cast"(%0) : (!verona.U64) -> !verona.S64
-    %2 = "verona.test"(%1) : (!verona.S64) -> !type<"U64 & imm">
-    return %2 : !type<"U64 & imm">
+  verona.class @C {
+  }
+  verona.class @D {
+    verona.field "f" : !verona.U64
+    verona.field "g" : !verona.U64
```

Comment:
The previous test checks both `U64` and `S64`. We should keep both in the new test, too.

Reply:
Done.